util/guard-one/guard-one.pact

```diff
@@ -0,0 +1,99 @@
+(namespace 'util)
+
+(module guard-one AUTONOMOUS
```

Comment:
let's call this `guards1` instead to indicate its relationship to the already-existing `guards` module and to also keep it brief.

Reply:
Changed